install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -328,8 +328,11 @@ data:
 {{- if .Values.global.kubeConfigPath }}
   k8s-kubeconfig-path: {{ .Values.global.kubeConfigPath | quote }}
 {{- end }}
-{{- if not (eq .Values.global.cni.chainingMode "none") }}
-  # Chaining mode was enabled, disabling health checking
+{{- if eq .Values.global.cni.chainingMode "portmap") }}
+  # Chaining mode is set to portmap, enable health checking
+  enable-endpoint-health-checking: "true"
+{{- else }}
```

Comment:
This will now disable endpoint health checking if `chainingMode = none` which is wrong.

Reply:
Thanks for your input, may be I misunderstood this, so if chainingMode is 'none' should we should enable 'endpoint-health-checking'?
Will address your comments.